tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -30,3 +31,9 @@ def get_resource_type(resource_id):
     @staticmethod
     def get_resource_name(resource_id):
         return resource_id.split('/')[8]
+
+
+def utcnow():
```

Comment:
a one line function for a one liner, feels a bit odd, though this is just a move.

Reply:
To make patching/mocking utcnow() easier I'm guessing?

Reply:
oic, you just moved this - yes it is used in some unit tests where `now` needs patching - having a wrapper method makes it cleaner.